packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -329,6 +331,18 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 			uploadTime: pending?.uploadTime,
 		});
 	}
+
+	public hasPendingStashedBlobs(): boolean {
+		if (
+			Array.from(this.pendingBlobs.values()).filter((e) => e.uploading === true).length > 0 &&
+			this.stashing
+		) {
+			this.stashing = false;
```

Comment:
Should this be cleared here? What if the container disconnects and reconnects before we finish uploading stashed blobs?